lld/ELF/Driver.cpp

```diff
@@ -574,6 +576,38 @@ static GcsPolicy getZGcs(Ctx &ctx, opt::InputArgList &args) {
   return ret;
 }
 
+static GcsReportPolicy
+getZGcsReport(Ctx &ctx, opt::InputArgList &args, bool isReportDynamic,
```

Comment:
I think the code will be cleaner if we compute the two values together in this function.